cinema/tests/test_movie_api.py

```diff
@@ -157,3 +158,164 @@ def test_image_url_is_shown_on_movie_session_detail(self):
         res = self.client.get(MOVIE_SESSION_URL)
 
         self.assertIn("movie_image", res.data[0].keys())
```

Comment:
The test `test_image_url_is_shown_on_movie_session_detail` checks for 'movie_image' in the response data. Ensure that the API returns the correct key for the image URL in the movie session detail response.